sktime/forecasting/model_selection/_split.py

```diff
@@ -834,6 +835,9 @@ def __init__(
         self.step_length = step_length
         self.start_with_window = start_with_window
         self.initial_window = initial_window
```

Comment:
please, move this line to the else statement that is now absent from the if statement below. and check for `initial_window`, not `self.initial_window`